File: Kattis/nastyhacks/nastyhacks.cpp
#include <iostream>

int main() {
    int n;
    std::cin >> n;

    while (n > 0) {
        int r, e, c;
        std::cin >> r >> e >> c;

        if (e - c > r) {
            std::cout << "advertise\n";
        }
        else if (e - c < r) {
            std::cout << "do not advertise\n";
        }
        else {
            std::cout << "does not matter\n";
        }
        n--;
    }
    return 0;
}

File: Kattis/nastyhacks/nastyhacks.py
n = int(input())

while n > 0:
    r, e, c = input().split()
    r = int(r)
    e = int(e)
    c = int(c)

    if e - c > r:
        print("advertise")
    elif e - c < r:
        print("do not advertise")
    else:
        print("does not matter")
    
    n -= 1